Makefile.am

```diff
@@ -1,3 +1,3 @@
-ACLOCAL_AMFLAGS = -Im4
-EXTRA_DIST = autogen.sh
-SUBDIRS = steam
+ACLOCAL_AMFLAGS = -I m4 ${ACLOCAL_FLAGS}
```

Comment:
Is a relict from me messing around with different settings and getting gtk-doc to run.